packages/adapters/src/process-instance-adapter.ts

```diff
@@ -72,15 +75,23 @@ class ProcessInstanceAdapter implements
             this.logger.trace(gotPython);
             const runnerPath = path.resolve(__dirname, require.resolve("@scramjet/python-runner"));
 
+            if (this.sthConfig.debug)
+                debugFlags = ["-m", "pdb", "-c", "continue"];
+
             return [
                 "/usr/bin/env",
                 "python3",
+                ...debugFlags,
                 path.resolve(__dirname, runnerPath),
                 "./python-runner-startup.log",
             ];
         }
+        if (this.sthConfig.debug)
+            debugFlags = ["--inspect=9229", "--inspect-brk"];
```

Comment:
`--inspect-brk=9229` would do the same job ;)

Reply:
Thanks, [changed](https://github.com/scramjetorg/transform-hub/pull/808/commits/7d2ba90c5d4b7c67df04d6c26f687e24fa29990a)!